app/src/main/java/com/infomaniak/drive/ui/MainViewModel.kt

```diff
@@ -196,10 +195,11 @@ class MainViewModel(appContext: Application) : AndroidViewModel(appContext) {
         val apiResponse = ApiRepository.moveFile(file, newParent)
         if (apiResponse.isSuccess()) {
             FileController.getRealmInstance().use { realm ->
+                file.getStoredFile(getContext())?.let { if (it.exists()) it.delete() }
                 FileController.removeFile(file.id, recursive = false, customRealm = realm)
                 FileController.updateFile(newParent.id, realm) { localFolder ->
                     file.isOffline = false
-                    localFolder.children.add(file)
+                    runCatching { localFolder.children.add(file) }
```

Comment:
I think we should add the issue link as a comment, above this line.

`https://github.com/Infomaniak/android-kDrive/issues/503`